addons/docs/src/blocks/DocsContainer.tsx

```diff
@@ -75,11 +75,11 @@ export const DocsContainer: FunctionComponent<DocsContainerProps> = ({ context,
         setTimeout(() => {
           scrollToElement(scrollTarget, 'start');
         }, 200);
+      } else if (docsWrapperRef.current) {
```

Comment:
Also moved this inside the else block so it only runs when no element is found instead of if the ref exists.